core/src/main/java/org/elasticsearch/search/suggest/Suggest.java

```diff
@@ -553,7 +558,7 @@ public Option(Text text, Text highlighted, float score) {
                 }
 
                 public Option(Text text, float score) {
-                    this(text, null, score);
+                    this(text, null, score, null);
```

Comment:
Sure, will change. But out of curiosity, what is wrong with directly calling the ctor directly?